lib/base16.rb

```diff
@@ -0,0 +1,10 @@
+# See https://www.rfc-editor.org/rfc/rfc4648#section-8
+class Base16
+  def self.encode16(str)
+    str.unpack1('H*').tap(&:upcase!)
+  end
+
+  def self.decode16(str)
+    [str].pack('H*')
+  end
+end
```

Comment:
The nittiest of nits; GitHub gets angry if there's not a newline at the end of the file.